### ISSUE TITLE
add option to override phoenixwallet.me bip353 address

### ISSUE THREAD

**Issue by martindmtrv** [ACINQ/phoenix#597]
I just messed around with the new bolt12 support and it's awesome! I dug into the bip a bit more and found out how to create my own address with my domain DNS that I own. 

I'm hoping that we can add an option to override the @phoenixwallet.me address if we have our our domain that we added a valid DNS entry for. That way it will show our own custom address / domain in the receive tab instead of the auto generated one

Not sure how the ux for this should look, maybe a user will enable the custom address and on adding one it should verify that the address does actually resolve to the users bolt12 offer before saving it!

**Comment by dpad85** [comment]
Good idea, that's something we could add in the experimental feature section for now.

>  it should verify that the address does actually resolve to the users bolt12 offer before saving it!

good point!
